shell/platform/android/context/android_context.h

```diff
@@ -16,6 +16,9 @@ enum class AndroidRenderingAPI {
   kSoftware,
   kOpenGLES,
   kVulkan,
+  /// @brief Attempt to create a Vulkan surface, if that fails then fall back
+  /// to GLES.
+  kAutoselect,
```

Comment:
I don't know if adding this will cause issues elsewhere, I''m going to run it on CI and see.

Alternatively could just use a separate flag below.